### ISSUE TITLE
Taking advantage of duplicate label

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#596]
On Webcompat there are the bug 809 and 819. Both are exactly the same issue.
When we close 819 with a duplicate, we should make it visible in 809 that there's a duplicate.

Maybe the condition is closed + duplicate flag.

Issue: There is no tied notion of duplicate + bug_number. So the risk is that any references and duplicate label will generate a link (this is not cool).

Maybe just impossible in the way github is made. 
Maybe manageable on the side of webcompat.com. Aka when we change the label as duplicate the UI is asking us which number.

TO THINK. :thought_balloon:

**Comment by miketaylr** [comment]
Yeah, this is a feature that GitHub totally fails on.

> So the risk is that any references and duplicate label will generate a link (this is not cool).

Can I ask what you mean here? Not sure I understand.

**Comment by karlcow** [comment]
> Can I ask what you mean here? Not sure I understand.

sure. If we check for is there `#000` number and a label `invalid`. Each time, someone will have cited another bug not only the duplicate bug, but any bugs such as a "see also blah", it will trigger the association. Because we have no way to know it's the right thing.

**Comment by hallvors** [comment]
If we have a UI for requesting a master bug number on webcompat.com, the webcompat bot can add a specially formatted comment ("This bug is closed as a duplicate of #000") which we can look for to create the right link. This has drawbacks because localization would mess up the JS looking for a specific English string, but I can't think of any better approach right now.

**Comment by miketaylr** [comment]
That seems doable. A few other crazy ideas (that I'm not convinced are great, but are maybe options):

* create a `duplicate_of_N` label programatically, but then you end up with a billion rarely used labels. We can hide these labels in the webcompat UI, but still seems gross.
* Add an html comment in the bug description `<!-- dupe: 222 -->`.

After typing this out, I think a regular comment is the least crazy.

**Comment by miketaylr** [comment]
Probably depends on #1305. But I agree we want this.

**Comment by karlcow** [comment]
https://github.com/blog/2398-better-manage-issues-and-pull-requests-with-keyword-updates

` duplicate of #`
> To flag a duplicate, add a comment using the duplicate of keyword followed by the issue number or URL. A "Marked as duplicate" timeline event will appear in the timelines of the referenced issues.

Interesting to dig further.

**Comment by karlcow** [comment]
An example.
This is the list of all events for the issue 8684
https://api.github.com/repos/webcompat/web-bugs/issues/8684/events

As we can see, there are the usual "labeled" events and a "marked_as_duplicate" event which gives an URI.
https://api.github.com/repos/webcompat/web-bugs/issues/events/1194983872

💩 weird… because they don't take full advantage of it. They lost the opportunity to give also the number of the bug which we duplicate against.
